src/index.js

```diff
@@ -356,9 +356,24 @@ class IPLDResolver {
  * @returns {void}
  */
 IPLDResolver.inMemory = function (callback) {
+  const Memdown = require('memdown')
   const repo = new IPFSRepo('in-memory', {
-    fs: MemoryStore,
-    level: require('memdown'),
+    storageBackends: {
+      root: MemoryStore,
+      blocks: MemoryStore,
+      datastore: MemoryStore
+    },
+    storageBackendOptions: {
+      root: {
+        db: Memdown
+      },
+      blocks: {
+        db: Memdown
+      },
+      datastore: {
+        db: Memdown
+      }
+    },
```

Comment:
@diasdavid looks like it. I think that, for the tests, it doesn't make a difference whether we use an extension or sharing. Wouldn't you agree?